src/menu.c

```diff
@@ -36,6 +56,7 @@ void show_menu() {
 
 /** @brief Function parses user input (single characters) on @c stdin */
 void parse_input() {
```

Comment:
Im Grunde wurden die Programm blöcke für die jeweilige Applikation vom testing.c in die switch cases kopiert und eingefügt